src/components/atoms/UiTextarea/UiTextarea.vue

```diff
@@ -80,10 +102,45 @@ const inputHandler = (event: Event) => {
   emit('update:modelValue', el.value);
 };
 const resizeValue = computed(() => {
+  if (props.hasAutogrowing) {
+    return 'none';
+  }
   if (typeof props.resize !== 'boolean') {
     return props.resize;
   }
-  return props.resize ? 'both' : 'none';
+  return props.resize
+    ? 'both'
+    : 'none';
+});
+const textarea = ref<HTMLTextAreaElement | null>(null);
+const textareaSize:Size = reactive({
+  width: null,
+  height: null,
+});
+const setTextareaSize = (mutationList: MutationRecord[]) => {
+  const { style } = mutationList[0].target as HTMLElement;
```

Comment:
IMO this assertion should also be replaced with proper type-narrowing. At least unless you're 100% sure that it's an HTMLElement - then the most important question is - Why does TypeScript think differently?

Reply:
@AlanLes I tried to find another solution, but I can't and also I find this [link](https://github.com/microsoft/TypeScript/issues/47155) with the same problem. Maybe do you suggest better solution?

Reply:
Simply this one line should assure TypeScript that the `style` indeed exists in the element you take it from :) 

```
if (!('style' in mutationList[0].target)) throw new Error('This is not a valid HTMLElement');
const { style } = mutationList[0].target;
...
```

Reply:
It's always good to first check if the property you expect really exists on some object and throw an error otherwise. If **yes**, then you've satisfied TS and no need to do assertion - **win** 👍  Otherwise - you've found a spot where your application may crush, because the type is not what you expected (and wanted to cast with `as`) 😝

Reply:
@AlanLes this solution change type :
![image](https://github.com/infermedica/component-library/assets/37744371/a12183ab-15bf-4bdc-b97b-8a193de1c413)

Reply:
Yes, it probably does. It looks like the MutationRecord is either wrongly typed or its usage here is a mistake, which should be done in another way.
On the other hand, the `MutationRecord[]` is an Array of `MutationRecord` type elements. The `target` property of the `MutationRecord` is a type of `Node`. So it looks like there's no `style` key on the `Node` element, and that's why TS doesn't know anything about its' `width` and `height` keys. So it should be guarded the same way as we did for `style` :)

Reply:
Anyway, the `assertion` is still on the table - it exists in the TypeScript for some reason, and it should be used, yet should be used only as a last resort. If you think this is the last resort - do it. 😄